aiida_quantumespresso/workflows/pw/base.py

```diff
@@ -168,18 +168,17 @@ def get_builder_from_protocol(
         nkinds = len(structure.kinds)
 
         try:
-            types = (UpfFamily, SsspFamily)
-            pseudo_family = orm.QueryBuilder().append(types, filters={'label': pseudo_family}).one()[0]
+            pseudo_family = orm.QueryBuilder().append(SsspFamily, filters={'label': pseudo_family}).one()[0]
         except exceptions.NotExistent as exception:
             raise ValueError(f'required pseudo family `{pseudo_family}` is not installed') from exception
```

Comment:
Maybe it would be good to give the new user a little hand here, and point them towards installing the default SSSP family with `aiida-pseudo`.

Reply:
Done